docs/source/user.rst

```diff
@@ -0,0 +1,254 @@
+
+.. note:: Any non-empty string can be used as an event identifier.
+
+Batches can be created from the recipe's change page by pressing the
+'Create batch from recipe' button.  Placeholder values for the
+batch-specific information must be changed immediately to avoid
+conflict with other batches.
+
+.. todo:: There is an open issue about this problem.
+
+Products
+~~~~~~~~
+
+Products are batches which have been bottled.  Once the bottling is
+complete, labels can be generated based on the product's ingredients
+and other characteristics.  Jars of products that have been labeled
+can be entered into the inventory management system.
+
+Product names, titles, and descriptions are all inherited from their
+batches.  Product titles and descriptions are currently used for both
+labels and distribution.
+
+When a batch is bottled into jars, the Jars field of the batch's
+change page must be updated and the batch saved.  At this point labels
+can be printed and applied to the jars.  A product can then be created
+from that batch by pressing the 'Create product from batch` button on
+the batch's change page.
+
+Samples
+~~~~~~~
+
+Samples are taken throughout the lifetime of the batch.  The following
+information is collected for each sample:
+
+* Temperature (in degrees Fahrenheit)
+* Specific gravity
+* Notes (traditionally used for sensory data)
+
+These samples are used to calculate the percent alcohol of the mead,
+and are convenient for storing information which can be referenced
+when making labels.
+
+Storage
+-------
+
+Once the mead is produced, it must be stored in such a manner as to
+facilitate easy retrieval of specific jars.  This requires an
+inventory management system.  The largest unit of storage in the IMS
+is the warehouse.  Each warehouse is a collection of rows.  Each row
+is a collection of shelves.  Each shelf is a collection of crate-sized
+bins.  Each bin contains one or more crates.  Each crate contains one
+or more jars.  At least one bin and one crate must be created for the
+IMS to function.
+
+Create jars
+~~~~~~~~~~~
+
+To actually create the jars requires a management command.  Here is an example::
+
+  (venv)# python manage.py add_new_jars --product="SIP 26 C" --start-jar=1 --end-jar=12 --crate=37
+  12 jars were created in SIP 26 C and placed in Crate 37
+
+The crate must exist and have sufficient capacity to contain the jars
+for this command to succeed.  At this point the jars are now available
+for distribution.
+
+.. note:: If the producer wishes to protect a jar from being
+          distributed, they should use the admin interface to set that
+          jar's "is available" flag to False.
+
+Crate utilization and transfer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Over time, crates will become partly empty and will need to be
+consolidated in order to make empty crates available for new product.
+Crate utilization is reported with a management command.  Here is an
```

Comment:
I know what you mean by crate utilization and reporting it, but if I walked in off the street I would need definitions...going from sentence starting with "Over time" to the sentence starting with "crate utilization" would be a leap. I would need something like "a report called crate utilization will tell the user how many jars are left in each crate. The report can be used to determine whether there are enough empty spaces to consolidate crates by transferring jars. Here's how to transfer jars." Etc.